RimWorld/RimWorld/main.cpp

```diff
@@ -0,0 +1,102 @@
+﻿#include <SFML/Window.hpp>
+#include <SFML/Graphics.hpp>
+#include <iostream>
+#include <algorithm>
+#include <thread>
+
+#include "Component.h"
+#include "Map.h"
+#include "Tool.h"
+#include "NpcManager.h"
+#include "ResourceManager.h"
+
+#include "WindowHelper.h"
+
+bool isPlacingMode = false;
+bool isZoningMod = false;
+bool isOrdersMod = false;
```

Comment:
Better use State Pattern.
Or, at least use enum class if it's not allowed to be in several states at once